src/pack.rs

```diff
@@ -235,6 +237,60 @@ pub async fn pack(options: PackOptions) -> Result<()> {
         bar.pb.finish_and_clear();
     }
 
+    let injected_pypi_packages: Vec<_> = options
+        .injected_packages
+        .iter()
+        .filter(|e| {
+            e.extension()
+                .filter(|e| e.to_str() == Some("whl"))
+                .is_some()
+        })
+        .cloned()
+        .collect();
+
+    tracing::info!("Injecting {} pypi packages", injected_pypi_packages.len());
+    for path in injected_pypi_packages {
+        let filename = path
+            .file_name()
+            .ok_or(anyhow!("cannot get filename"))?
+            .to_str()
+            .ok_or(anyhow!("could not convert filename to string"))?
+            .to_string();
+        let path_str = path
+            .to_str()
+            .ok_or(anyhow!("could not convert filename to string"))?
+            .to_string();
+        let wheel_file_name = WheelFilename::from_str(&filename)?;
+        let pypi_data = PypiPackageData {
+            name: wheel_file_name
+                .name
+                .as_str()
+                .parse()
+                .map_err(|e| anyhow!("cannot parse package name: {}", e))?,
+            version: wheel_file_name
+                .version
+                .to_string()
+                .parse()
+                .map_err(|e| anyhow!("cannot parse package version: {}", e))?,
+            location: path_str
+                .parse()
```

Comment:
Is this converting back to a path? If yes, why not use the original path?

Reply:
No, it's converting from `String` to `rattler_lock::UrlOrPath`.
`UrlOrPath` does not implement `From<PathBuf>`